src/App.js

```diff
@@ -1,13 +1,70 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      todos: [
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 1528817084358,
+          completed: false
+        }
+      ],
+      todo: ''
+    }
```

Comment:
I understand that `todo` here (inside your state) is supposed to hold the value of your input form. However, at first glance, it might not be easy to infer that from the name you gave.

Perhaps you could name this something like `todoInput` or `newTodo` instead to make its purpose clearer.